docs/book/src/component-config-tutorial/testdata/project/config/default/manager_metrics_patch.yaml

```diff
@@ -0,0 +1,18 @@
+# This patch adds the args to allow expose the metrics endpoint
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    metadata:
+      labels:
+        allow: allow-metrics
```

Comment:
I don't think this extra label is required. We should instead use existing labels addressing the manager pods. This comment explains how I think this should be configured overall: https://github.com/prometheus-operator/kube-prometheus/discussions/1907#discussioncomment-3896712. The network policy gives 0 security without the namespace selector selecting namespaces to access ingress from, which I think we should be extremely careful to not mislead our users by noting this as a "replacement". In my opinion, removing the opt-in kube-rbac-proxy and adding an opt-in for a network policy are two different things.

Reply:
Hi @erikgb,

Thank you for your feedback. Following the comments inline

Regards

> `The network policy gives 0 security without the namespace selector selecting namespaces to access ingress from, which I think we should be extremely careful to not mislead our users by noting this as a "replacement".`

**IMPORTANT: We are NOT saying that is a replacement at all.**
Please, see that is clarified with WARNING class in the docs and Proposal.
However, it is very important and we MUST to have the phase 1 scaffold in place. Otherwise we end up without no protection at all, and either without NO possibility to warn.

We cannot do a PR with ALL phases by once. it is too much changes to address then, because of this we agreed in the proposal to address it per phases. 

See that use ONLY network policies does not ensure a good protection at all. and we are clarifying it in the docs + proposal. I update the title of this PR if it is what brings the concern to you.

Regards the other raised points I will check each of them out and I will came back to you.

Reply:
Hi @erikgb 

Again, really thank you for all your feedback.
I tried to reply all above and see that from mainly the 4/5 items suggested by you we are addressing 3 :-)

See that I added what is missing get done here in the description.